packages/okta-react-native/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+#1.4.0
+
+### Features
+- [`63c0b89`](https://github.com/okta/okta-oidc-js/pull/751/commits/63c0b892ea1bea085ee594aab480e97c41969a32)
```

Comment:
You can just link to the PR (no need for the commit SHA)

Reply:
Also the link text can be the PR #

Reply:
Sure, I was just following the current changelog, will change.

Reply:
We used to do it the other way with the commits, but it's a hassle to update everytime you rebase. The link to the PR is better for users because they can read our interesting conversations.